### PR TITLE
Modifprofile

### PR DESCRIPTION
**Pull request by leolab69**

### CODE REVIEW

**Review by ChrichriR — APPROVED** [review]

File: model/PractitionersData/practitionersData.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { child, get, ref } from "firebase/database";
+
+export async function getAllPractitionersData() {
+  const user = auth.currentUser;
+  if (user) {
+    try {
+      const snapshot = await get(child(ref(db), `/practitioners/${user.uid}`));
+      if (snapshot.exists()) {
+        return snapshot.val();
+      } else {
```
Comment (NycoChavanoud) [inline model/PractitionersData/practitionersData.js:11]: console.log?

File: pages/profile/editPractitioner.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+import styles from "./modif-profile.module.css";
+import AddCircleIcon from "@mui/icons-material/AddCircle";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { update, ref } from "firebase/database";
+import { db, auth } from "../../config/firebaseConfig";
+import { getAllPractitionersData } from "../../model/PractitionersData/practitionersData";
+
+export default function Profile() {
+  const [practitionersData, setPractitionersData] = useState();
```
Comment (NycoChavanoud) [inline pages/profile/editPractitioner.js:15]: init state

File: pages/profile/editPractitioner.js [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+      router.push("/profile");
+    } catch (error) {
+      setError("erreur");
+    }
+  };
+
+  useEffect(() => {
+    getAllPractitionersData().then(setPractitionersData);
+    console.log(practitionersData);
```
Comment (NycoChavanoud) [inline pages/profile/editPractitioner.js:36]: clean

File: pages/profile/editPractitioner.js [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+              }
+            />
+          </div>
+
+          <div className={styles.field}>
+            <label className={styles.label} htmlFor="phone"></label>
+            <input
+              data-cy="phone"
+              className={styles.input}
+              type="tel"
```
Comment (NycoChavanoud) [inline pages/profile/editPractitioner.js:109]: revoir setup input phone

File: pages/profile/index.js [hunk 5]
```diff
@@ -1,62 +1,86 @@
...
-import Image from "next/image";
 import Link from "next/link";
 import style from "./profile.module.css";
+import { useEffect, useState } from "react";
+import { getAllPractitionersData } from "../../model/PractitionersData/practitionersData";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 
 export default function Profile() {
+  const [practitionersData, setPractitionersData] = useState();
```
Comment (NycoChavanoud) [inline pages/profile/index.js:12]: init

File: pages/profile/index.js [hunk 6]
```diff
@@ -1,62 +1,86 @@
...
+
+  useEffect(() => {
+    getAllPractitionersData().then(setPractitionersData);
+    console.log(practitionersData);
```
Comment (NycoChavanoud) [inline pages/profile/index.js:16]: clean

File: pages/profile/index.js [hunk 7]
```diff
@@ -1,62 +1,86 @@
...
-              src="/img/fb.png"
-              alt="logo facebook"
-            />
-            <p className={style.lnk}>@Sophrologue</p>
-          </a>
-        </Link>
+        <a
+          target="_blank"
+          rel="noreferrer"
+          href={practitionersData?.fb_url}
```
Comment (NycoChavanoud) [inline pages/profile/index.js:46]: revoir link

**Review by NycoChavanoud — APPROVED** [review]

**Pull request merged**